### ISSUE TITLE
Feature Request: Making all GPUs on rig switch Algo's together

### ISSUE THREAD

**Issue by Jesso2k** [angelbbs/NiceHashMinerLegacy#176]
I know we can group same GPU's together for profit switching but it there a way we can force all GPU's to a rig to switch together?

I ran into an issue where my x3 3070s were switching to CuckooCycle on Gminer, but the rest of my GPU's were on Dagger with NBMiner. For whatever reason running both was causing a conflict and crashing out NBMiner when it's normally rocksolid for weeks at a time.

Thanks

**Comment by angelbbs** [comment]
change hashrate on benchmark page for some gpu

**Comment by angelbbs** [comment]
*settings->devices/algoritms page

**Comment by Jesso2k** [comment]
Oh neat. So if we were to total the hashrate for each algo, average it out and input it manually into that box plus power in the other, it'll profit switch with the behaviour I desire?

I'll try it out, thanks for an awesome utility.

**Comment by angelbbs** [comment]
yes, power is also better to average if you have entered the cost of electricity
